components/mosart/cime_config/testdefs/testmods_dirs/mosart/rof_ocn_2way/user_nl_mosart

```diff
@@ -0,0 +1,4 @@
+frivinp_rtm = '$DIN_LOC_ROOT/rof/mosart/MOSART_2way_LLR3_c220615_coupling_test.nc'
+routingmethod = 2
```

Comment:
The value of routingmethod can be revised in RtmMod.F90. So instead of using two flags to turn on this feature, I'd suggest to use only one flag: use_ocn_rof_two_way. You can add some lines in RtmMod to check whether use_ocn_rof_two_way is true, and if true, make routingmethod equal to DW.

Reply:
Thanks. The modification was made in RtmMod.F90 to force routingmethod=DW when the use_ocn_rof_two_way flag is on.